plugins/woocommerce/src/Blocks/BlockTypes/ProductGallery.php

```diff
@@ -135,6 +135,10 @@ protected function render( $attributes, $content, $block ) {
 						'productId'         => $product_id,
 						'selectedImageId'   => $initial_image_id,
 						'userHasInteracted' => false,
+						'overflowTop'       => false,
+						'overflowBottom'    => false,
+						'overflowLeft'      => false,
+						'overflowRight'     => false,
```

Comment:
Do you think that prefixing those with `thumbnails` would make sense?

Reply:
Yeah, context is "global" for whole Product Gallery while overflow is inner block specific. Good catch!